Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "CloudWatchLogs"
 uuid = "c4c1e6a2-6bdd-52e0-b56d-1d4734724d2d"
-version = "2.0.0"
+version = "2.0.1"
```

Comment:
Do I even need to bump the release?

Reply:
I was thinking we'd merge this so we could see what the Compat.toml was supposed to look like, but if we get the registry PR in first then no we don't need to bump the release.

Reply:
```suggestion
version = "2.0.0"
```